server.go

```diff
@@ -3839,6 +3839,7 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 		SigPool:                 s.sigPool,
 		Wallet:                  s.cc.Wallet,
 		ChainNotifier:           s.cc.ChainNotifier,
+		CurrentChainStateView:   s.cc.CurrentChainStateTracker,
```

Comment:
I think it also no longer needs the `ChainNotifier`?

Reply:
It seems to need it to wait for confirmations on channel closures it seems, still.